scripts/us_epa/ejscreen/ejscreen.py

```diff
@@ -108,32 +98,136 @@ def write_csv(data, outfilename):
 
 
 def write_tmcf(outfilename):
+    """
+    Writes a list of dictionaries (representing MCF nodes) to a file in MCF format.
+
+    Args:
+        outfilename: The name of the output file.
+    """
+    mcf_content = []
+
+    if isinstance(TEMPLATE_MCF, list):
+        for node in TEMPLATE_MCF:
+            if isinstance(node, dict):
+                lines = []
+                for key, value in node.items():
+                    # Handle multi-line values by joining them with a newline
+                    if isinstance(value, list):
+                        value_str = "\n".join(f"  {v}" for v in value)
+                        lines.append(f"{key}:\n{value_str}")
+                    else:
+                        lines.append(f"{key}: {value}")
+                mcf_content.append("\n".join(lines))
+    else:
+        # Assuming a single dictionary or string for non-list TEMPLATE_MCF
+        if isinstance(TEMPLATE_MCF, dict):
+            lines = [f"{key}: {value}" for key, value in TEMPLATE_MCF.items()]
+            mcf_content.append("\n".join(lines))
+        else:
+            mcf_content.append(str(TEMPLATE_MCF))
+
+    template_content = "\n\n".join(mcf_content)
+
     with open(outfilename, 'w') as f_out:
-        f_out.write(TEMPLATE_MCF)
+        f_out.write(template_content)
+
+
+def main(_):
+    global URL_SUFFIX, BASE_URL, TEMPLATE_MCF, FILENAMES
+
+    try:
+        # Load configuration from config.json
+        with file_util.FileIO(_FLAGS.config_path, 'r') as f:
+            config = json.load(f)
+
+        YEARS = config["YEARS"]
+        NORM_CSV_COLUMNS = config["NORM_CSV_COLUMNS"]
+        NORM_CSV_COLUMNS1 = config["NORM_CSV_COLUMNS1"]
+        CSV_COLUMNS_BY_YEAR = config["CSV_COLUMNS_BY_YEAR"]
+        ZIP_FILENAMES = config["ZIP_FILENAMES"]
+        FILENAMES = config["FILENAMES"]
+        TEMPLATE_MCF = config["TEMPLATE_MCF"]
+        BASE_URL = config["BASE_URL"]
+        URL_SUFFIX = config["URL_SUFFIX"]
+        RENAME_COLUMNS_YEARS = config["RENAME_COLUMNS_YEARS"]
+
+        dfs = {}
+        input_folder = os.path.join(_MODULE_DIR, 'input')
+
+        # Download files if the mode is 'download' or if no mode is specified
+        if _FLAGS.mode == "" or _FLAGS.mode == "download":
+            for year in YEARS:
```

Comment:
Can we have it process all years from 2015 to the current year instead of a fixed list?
That way, when a new year's data is available, it can fail until the config is updated to include column headers.

In the current mode, the lastDataRefreshDate will move to 2025 without 2025 data being added even if available and this will not be caught by automated staleness alerts.